dishes/colombian/otras_preparaciones/envueltos.md

```
@@ -1,108 +1,33 @@
+
+*   10 mazorcas tiernas y grandes
+*   1/2 taza de mantequilla derretida
+*   1/2 taza de azúcar (o al gusto)
+*   1/4 taza de leche o crema de leche
+*   1 cucharadita de polvo de hornear
+*   1/2 cucharadita de sal
+*   Queso campesino o cuajada en cubos (opcional)
+
+## Preparación
+
+1.  Con cuidado, retira las hojas de las mazorcas, tratando de mantenerlas enteras. Lávalas y resérvalas.
```

markdownlint-cli2 (0.17.2)

Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)